nrf_wifi/hw_if/hal/inc/fw/phy_rf_params.h

```diff
@@ -338,6 +338,17 @@ enum MAX_POWER_OFFSETS {
 	NRF_WIFI_MAX_OP_PWR_5GHZ_HI_MCS0
 };
 
+/** The byte offsets of RF parameters indicate the start offset
+ * of PCB loss for 2.4 GHz and 5 GHz bands.
+ */
+enum PCB_LOSS_BYTE_OFFSETS {
+	PCB_LOSS_BYTE_2G_OFST = 185,
+	PCB_LOSS_BYTE_5G_BAND1_OFST,
+	PCB_LOSS_BYTE_5G_BAND2_OFST,
+	PCB_LOSS_BYTE_5G_BAND3_OFST,
+	NUM_PCB_LOSS_OFFSET = 4
```

Comment:
This looks odd, is it really an offset? if yes, please add a comment, if not please move out of the enum

Reply:
These are offsets(185,186,187,188) indexed to access RF parameters, except the last one.

Reply:
Yes, my query was for the last one. So, it should be moved out of Enum.

FYI, even when you select a line and comment against that line, GH shows other lines for context, but the comment is applicable only for the last line.